FAQ.md

```diff
@@ -246,6 +246,31 @@ To run and train InstructLab locally, you must meet the following requirements:
 - Approximately 60GB of free disk space is needed to run the entire process locally on Apple hardware
 - About 32 GB RAM
 
+## Terms to Know
```

Comment:
nit: "Terms to Know" implies that folks noticing this section should internalize the terms you define in it.  That's a lot.  Can we call it "Glossary" or something that leaves things more open to both "I'll go memorize these terms" and "Ahh, I'll come back here if I bump into some jargon I want to unpack"?  Because personally my brain works much better with that kind of lazy-loading approach, when I have an actual use-case in mind driving me to try to figure out the word that I'm not yet familiar with ;)

Reply:
That's very nit :)

Reply:
I like the suggestion, fwiw

Reply:
Glossary is a helpful term because it is well understood what it means. I would go with this minor nit though very useful suggestion from @wking

Reply:
Personally I find it rather semantic, but not so much I won't go with how folks feel 😄 I will change this to "Glossary"